addons/mail/static/src/discuss/call/common/call_picture_in_picture.js

```diff
@@ -0,0 +1,144 @@
+import { Call } from "@mail/discuss/call/common/call";
+import { Component, useRef, useState, onMounted, onWillUnmount } from "@odoo/owl";
+import { registry } from "@web/core/registry";
+import { useService } from "@web/core/utils/hooks";
+
+const SYSTRAY_H = 40;
+
+export class CallPictureInPicture extends Component {
+    static props = [];
+    static components = { Call };
+    static template = "discuss.CallPictureInPicture";
```

Comment:
Would use acronyme PIP / pip in component name and file.
This is shorter so more readable, and it's pretty much mainstream to use it's shorter name "pip"